.github/workflows/carsus-data-repos-config.yml

```diff
@@ -0,0 +1,79 @@
+name: carsus-data-repos-config
+
+on: 
+  workflow_dispatch: 
+        
+env:
+  NBCONVERT_FLAGS: --execute --ExecutePreprocessor.timeout=600 --to html
+
+defaults:
+      run:
+        shell: bash -l {0}
+
+jobs:
+  carsus-build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: carsus/
+                      
+      - name: Cache carsus-data-cmfgen
+        uses: actions/cache@v3
+        with:
+          path: cloned-repo-carsus-data-cmfgen
+          key: ${{ runner.os }}-cache-${{ hashFiles('cloned-repo-carsus-data-cmfgen/**/*') }}
+        id: cmfgen-cache
+            
+      - name: Clone carsus-data-cmfgen
+        run: | 
+          git clone https://github.com/tardis-sn/carsus-data-cmfgen cloned-repo-carsus-data-cmfgen
```

Comment:
Lets consider this- if the cache was somehow restored, would this step then not download the entire thing? Or in different words- does the restore cache step work properly?

Reply:
Sorry, I did not understand what you meant here. Yes if the cache is there then this step is skipped.